chain.go

```diff
@@ -134,21 +134,20 @@ func (taskChain *TaskChain) Run() error {
 }
 
 // formatDuration is a time formatter.
-func formatDuration(d time.Duration) (s string) {
-	timeMsg := ""
+func formatDuration(d time.Duration) (timeMsg string) {
 
 	if d.Hours() > 1.0 {
-		timeMsg += fmt.Sprintf("%1.0fh", d.Hours())
+		timeMsg = fmt.Sprintf("%1.0fh", d.Hours())
 	}
 
 	if d.Minutes() > 1.0 {
-		timeMsg += fmt.Sprintf("%1.0fm", d.Minutes())
```

Comment:
True! This code was written half a year ago and I forgot how it worked. I added  a better comment explaining the format.